cmake/testing.cmake

```diff
@@ -138,7 +138,7 @@ if(VTK_VERSION VERSION_GREATER 9.0.1)
   f3d_test(NAME TestGLTFImporterUnlit DATA UnlitTest.glb)
   f3d_test(NAME TestMaterial DATA suzanne.ply ARGS --color=0.72,0.45,0.2 --metallic=1 --roughness=0.1)
   f3d_test(NAME TestMetaData DATA pdiag.vtu ARGS -m)
-  f3d_test(NAME TestInteractionAnimation DATA InterpolationTest.glb INTERACTION)#Space;Space;
+  f3d_test(NAME TestInteractionAnimation DATA InterpolationTest.glb ARGS --animation-index=-1 INTERACTION)#Space;Space;
```

Comment:
Why is it required? There is only one animation right?

Reply:
no they are many, for some reasons the diff is not catched by vtkImageDifference